src/storekit.cs

```diff
@@ -995,21 +1001,33 @@ interface SKAdNetwork {
 		[Static]
 		[Async]
 		[Export ("startImpression:completionHandler:")]
-		void StartImpression (SKAdImpression impression, [NullAllowed] Action<NSError> completion);
+		void StartImpression (SKAdImpression impression, [NullAllowed] SKAdNetworkCompletionHandler completion);
```

Comment:
This is a breaking change, you can only add a new overload, not change existing parameter types.